tpl/tplimpl/embedded/templates/shortcodes/x_simple.html

```diff
@@ -0,0 +1,60 @@
+{{- if not site.Config.Privacy.X.Disable -}}
+  {{- $id := or (.Get "id") "" -}}
+  {{- $user := or (.Get "user") "" -}}
+  {{- if and $id $user -}}
+    {{- template "render-simple-x" (dict "id" $id "user" $user "ctx" .) -}}
+  {{- else -}}
+    {{- errorf "The %q shortcode requires two named parameters: user and id. See %s" .Name .Position -}}
+  {{- end -}}
+{{- end -}}
+
+{{- define "render-simple-x" -}}
+  {{- $dnt := site.Config.Privacy.X.EnableDNT }}
+  {{- $url := printf "https://twitter.com/%v/status/%v" .user .id -}}
+  {{- $query := querify "url" $url "dnt" $dnt "omit_script" true -}}
+  {{- $request := printf "https://publish.twitter.com/oembed?%s" $query -}}
```

Comment:
Same as above.

Reply:
Other than that this looks and works great.

Reply:
Yeah, I was reticent to touch anything "under the hood" because I don't know if they really have their act together. The integration tests with the new domain all pass, so I've made the changes. 

But what's inserted in the HTML is still branded the old way...

```text
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Owl bet you&#39;ll lose this staring contest 🦉 <a href="https://t.co/eJh4f2zncC">pic.twitter.com/eJh4f2zncC</a></p>&mdash; San Diego Zoo Wildlife Alliance (@sandiegozoo) <a href="https://twitter.com/sandiegozoo/status/1453110110599868418?ref_src=twsrc%5Etfw">October 26, 2021</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Reply:
I still type in twitter.com in my browser from muscle memory ... that rebrand from twitter (a great name!) to x is the stupidest move, but if they don't have "their act together", I guess that's a clear signal that we need to retire that shortcode.